x-pack/plugins/security_solution/public/detections/pages/detection_engine/chart_panels/chart_select/helpers.test.ts

```diff
@@ -55,6 +64,7 @@ describe('helpers', () => {
           alertViewSelection,
           closePopover,
           setAlertViewSelection,
+          isAlertsPageChartsEnabled: true, // remove after charts is implemented
```

Comment:
Instead of putting reminders in the code, it's often better to make a ticket (if it doesnt already exist) and put the reminder there. That way we don't forget about it since we  will have a way to track it. 

In the code you could put a reference to the ticket number for easy lookup

Reply:
thank you! this is great advice - there is an issue for this feature, just added some clean up to-do's to address this.